### DISCUSSION TITLE
Does anyone have an idea how we can make it work in File Explorer?

### DISCUSSION THREAD

**Discussion by VincentLuders** [jcbastosportela/logioptspp#15] [General]
I tried running it, just to load logioptions, and then this app has an administrator.

**Comment by igvk** [comment]
Explorer doesn't support smooth scrolling in its windows, even if it receives high resolution scroll wheel events from the mouse.

**Comment by VincentLuders** [comment]
But, couldn't you not, in this case, let it appear as a trackpad scroll?

Best regards
*Vincent Lüders*
+352 661 610 666
